test/storage/2019-07-07/azblob/zz_generated_models.go

```diff
@@ -3214,6 +3214,8 @@ type ServiceGetUserDelegationKeyOptions struct {
 
 // ServiceListContainersSegmentOptions contains the optional parameters for the Service.ListContainersSegment method.
 type ServiceListContainersSegmentOptions struct {
+	// Include this parameter to specify that the container's metadata be returned as part of the response body.
+	Include *string
```

Comment:
This is weird, is this supposed to be a boolean? Just asking because of the comment.

Reply:
Nevermind, I see where it is used. I still think it's strange that it's not a boolean.

Reply:
I agree, and originally my thought was to change these to `bool` as that's how they behave.  However I wanted to get some feedback on how it currently works from the storage team.

Reply:
Well, if it's how it's defined in the swagger, I think it's the storage team who should decide to change this or not in the convenience layer. At the moment we arent adding any more magic for it, which I think is good.